src/io/parsers/CIFParser.js

```diff
@@ -60,753 +59,755 @@ function nameToElement(name) {
   // characters, however, it must start in column 13 even if the element
   // symbol is a single character (for example, see Hydrogen Atoms).
 
-  var veryLong = name.trim().length === 4;
+  const veryLong = name.trim().length === 4;
   return name.slice(0, veryLong ? 1 : 2).trim();
 }
 
-function CIFParser(data, options) {
-  Parser.call(this, data, options);
-  this.asymDict = {};
-  this.molecules = [];
-  this._options.fileType = 'cif';
+class AtomDataError extends Error {
+  constructor(message) {
+    super();
+    this.name = 'AtomDataError';
+    this.message = message;
+  }
 }
+//AtomDataError.prototype = Object.create(Error.prototype);
```

Comment:
Should we remove the commented line?

Note that such inheritance didn't work correclty in Babel 6 (see https://stackoverflow.com/questions/33870684/why-doesnt-instanceof-work-on-instances-of-error-subclasses-under-babel-node/33877501#33877501). I've updated to Babel 7 yesterday so it should now work correctly (https://babeljs.io/blog/2018/08/27/7.0.0#class-c-extends-htmlelement).

Reply:
After your update i remove this line with a clear conscience.